crates/nu-command/tests/commands/redirection.rs

```diff
@@ -126,6 +128,32 @@ fn separate_redirection() {
     )
 }
 
+#[test]
+fn same_target_redirection_with_too_much_stderr_not_hang_nushell() {
+    use nu_test_support::fs::Stub::FileWithContent;
+    use nu_test_support::pipeline;
+    use nu_test_support::playground::Playground;
+    Playground::setup("external with many stderr message", |dirs, sandbox| {
+        let bytes: usize = 81920;
+        let mut large_file_body = String::with_capacity(bytes);
+        for _ in 0..bytes {
+            large_file_body.push('a');
+        }
+        sandbox.with_files(vec![FileWithContent("a_large_file.txt", &large_file_body)]);
+
+        nu!(
+            cwd: dirs.test(), pipeline(
+            r#"
+            sh -c "cat a_large_file.txt 1>&2" out+err> another_large_file.txt
```

Comment:
I don't think this will work on native Windows. CI will probably be okay but anyone working on Windows will have a break with this test.

Can you write it differently so it doesn't use `sh`?